tests/e2e_tests/test_find.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from playwright.sync_api import Page, expect
+
+from tests.e2e_tests.helpers import login_via_magic_link, lookup_find_download_link_for_user_in_govuk_notify
+from tests.e2e_tests.pages.find import DownloadDataPage, FindRequestDataPage
+
+pytestmark = pytest.mark.e2e
```

Comment:
Could this global be set in `__init__.py` so it applies to all future E2E test modules?